python_bindings/apps/bilateral_grid.py

```diff
@@ -102,7 +102,7 @@ def generate_compiled_file(bilateral_grid):
 
 
 def get_input_data():
-    image_path = os.path.join(os.path.dirname(__file__), "../../apps/images/rgb.png")
+    image_path = os.path.join(os.environ["TEST_IMAGES_DIR"], "rgb.png")
```

Comment:
Should this really throw when not set? Why shouldn't it default to the old hardcoded value? Same with everywhere else that now reads an env-var.

At the very least, checking for a `KeyError` early on and printing a helpful error message would be nicer than "BZZT uncaught KeyError!"

Reply:
Because it should always be set.

Reply:
@abadams - I raised your same concern here.